scans/zapscan.py

```diff
@@ -0,0 +1,188 @@
+from time import sleep
```

Comment:
How would you feel about moving this script to either of the `devops` or `securedrop/bin` directories?